src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -235,6 +237,32 @@ class Preview extends React.Component<Props> {
         return this.iframeRef.contentDocument;
     };
 
+    getPreviewWindow= (): ?Window => {
+        if (this.previewWindow) {
+            return this.previewWindow;
+        }
+
+        if (!(this.iframeRef instanceof HTMLIFrameElement)) {
+            return;
```

Comment:
should here not the window returned and so in one case the iframe scroll position read and set and in other case the window scroll position?